MetriCam2/Camera.cs

```diff
@@ -683,7 +683,10 @@ public bool IsValid(object value)
                     { /* empty */ }
                     catch (OverflowException)
                     { /* empty */ }
+
+                    isTypeConvertible = (null != castedValue);
```

Comment:
@f00f ja, stimmt! Ich hatte übersehen, dass `isTypeConvertible` false initialisiert wird. Dann sollte es auch kein Prolem sein, wenn die Zuweisung in Falle einer Exception übersprungen wird.